CHANGELOG.md

```diff
@@ -4,58 +4,62 @@
 
 - Chg #161: Allow to reuse manager test code in related packages (@arogachev)
 - New #161: Add `ManagerInterface` (@arogachev)
-- Chg #161: Raise PHP version to 8.0 (@arogachev)
+- Chg #161, #217: Raise PHP version to 8.1 (@arogachev)
 - Bug #178: Exclude parent role from `Manager::getAllChildRoles()` (@arogachev)
 - Enh #134: Improve handling and control of `Assignment::$createdAt` (@arogachev)
 - Chg #134: Add `$createdAt` parameter to `ManagerInterface::assign()` (@arogachev)
-- Chg #134: Replace parameters with `$assignment` parameter in `AssignmentsStorageInterface::add()` (@arogachev)
-- Enh #165: Improve performance (@arogachev)
+- Chg #134: Replace all parameters with single `$assignment` parameter in `AssignmentsStorageInterface::add()` 
+  (@arogachev)
+- Enh #165, #206: Improve performance:
+  - Optimize calls for getting child items within the loops.
+  (@arogachev)
 - Enh #165: Rename `getChildren` method to `getDirectAchildren()` in `ItemsStorageInterface` (@arogachev)
-- Enh #165: Add methods to `ItemsStorageInterface`:
+- Enh #165, #203, #206, #208, #237: Add methods to `ItemsStorageInterface`:
     - `roleExists()`;
     - `getRolesByNames()`;
     - `getPermissionsByNames()`;
     - `getAllChildren()`;
     - `getAllChildRoles()`;
     - `getAllChildPermissions()`;
     - `hasChild()`;
-    - `hasDirectChild()`.
+    - `hasDirectChild()`;
```

Comment:
Need check correctly work of Yii Dev Tool with lists in changelog

Reply:
https://github.com/yiisoft/yii-dev-tool/issues/290

Reply:
For now, I decided to not use lists at all.